### PR TITLE
(feat): Add More infra experiments in the pipeline

### PR DESCRIPTION
**Pull request by uditgaurav**
Signed-off-by: udit uditgaurav@gmail.com

-  Add More infra experiments in the pipeline
   - node-drain 
   - disk-fill

[View Pipeline](https://gitlab.com/uditgaurav1/litmus-2e2-testing/pipelines/109263417)

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -62,6 +62,19 @@ Liveness:
     paths:
       - .kube/
 
+Auxiliary App:
+
+  stage: deploy
+  tags:
+    - setup
+  script:
+    - make auxiliaryapp
```
Comment (chandankumar4) [inline .gitlab-ci.yml]:
```suggestion
    - make auxiliary-app
```

File: Makefile [hunk 2]
```diff
@@ -33,6 +33,14 @@ liveness:
...
 	@ansible-playbook nginx/liveness/liveness.yml -vv
 	@kubectl get pods -n litmus
+
+.PHONY: auxiliaryapp
+auxiliaryapp:
+
+	@echo "------------------------"
+	@echo "Deploying Auxiliary App"
+	@echo "------------------------"
+	@kubectl run mypod --image=nginx -n litmus
```
Comment (chandankumar4) [inline Makefile]:
```suggestion
	@kubectl run nginx --image=nginx -n litmus
```

File: experiments/infra/disk-fill.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
+- hosts: localhost
+  vars:
+    a_ns: litmus
+    a_kind: deployment
+    name: engine-nginx9
```
Comment (chandankumar4) [inline experiments/infra/disk-fill.yml]: is the `9` added intentionally?

File: Makefile (hunk 2 above)
Comment (uditgaurav) [reply Makefile]: nginx in general app I'm deploying another app which is aux app

File: .gitlab-ci.yml (hunk 1 above)
Comment (uditgaurav) [reply .gitlab-ci.yml]: should I change this ?

Comment (chandankumar4) [reply .gitlab-ci.yml]: yes

File: Makefile (hunk 2 above)
Comment (chandankumar4) [reply Makefile]: you can set the name as `aux-app`

File: experiments/infra/disk-fill.yml (hunk 3 above)
Comment (chandankumar4) [reply experiments/infra/disk-fill.yml]: okay, for now, we can use some randomly generated string.

File: Makefile (hunk 2 above)
Comment (uditgaurav) [reply Makefile]: done

**Pull request merged**